vars/githubPr.groovy

```diff
@@ -15,32 +15,61 @@
 */
 def withDefaultPrComments(closure) {
   catchErrors {
+    buildState.set('PR_COMMENTS_ENABLED', true)
```

Comment:
Is there a reason we don't use `isPr() && params.ENABLE_GITHUB_PR_COMMENTS` instead of `true` here?

Reply:
That's better, I just changed it. I also just got rid of `params.ENABLE_GITHUB_PR_COMMENTS` since it's not actually needed anymore. It was left over from the GHPRB transition